docs/Documentation/Configuration.md

```diff
@@ -0,0 +1,80 @@
+
+- **Name:** A name for this group of settings.
+    
+- **Server:** Name or IP address of the mail server machine you want to connect. Please note that the Server address is only required for IMAP and POP3 connections. For Maildir and Localdir, there is no server needed since these refer to local directories.
+    
+- **Username:** The username or email address used to access the mailbox.
+    
+- **Password:** The password used to access the mailbox
+    
+- **Protocol:** You can select the type of protocol to be used for polling (IMAP, Maildir, POP3, Localdir).
+    
+- **Source:** Folder or path to poll from. For IMAP/POP3, this is typically "INBOX." For Maildir or Localdir, this is the directory path.
+    
+- **Use SSL?:** Whether to use SSL (Secure Sockets Layer) for IMAP and POP3 or not. SSL ensures that the connection to the mail server is encrypted and secure. For Maildir and Localdir, there is no need for SSL settings, since they establish no network connection.
+    
+- **Sender Group:** The group of contacts allowed to send mailings using these MailSettings. Only emails from contacts in this group will be processed.
+    
+- **Recipient Group:** The group of contacts to receive mailings using these MailSettings. Only contacts in this group will be sent mailings.
+    
+
+The **Sender Group** and **Recipient Group** should consist of CiviCRM contact groups. This feature ensures that only authorized individuals can send emails and that specific groups receive the messages. This allows for effective management of incoming and outgoing emails.
+
+- **Subject:** A pattern to use as the subject for the Mailing. Here you can use Tokens to automatically the required data.
+    
+- **From email address:** The from email address to use when sending notifications.
+    
+- **Notify disallowed sender:** When this setting is enabled and an email is received from a sender who is disallowed, the system will send a notification to that sender informing them that their email was not accepted or processed.
+    
+- **Disallowed sender notification subject:** The subject to use for notifications sent to disallowed senders.
+    
+- **Template for notifications of disallowed sender:** A Smarty template to use for notifications sent to disallowed senders.
+    
+- **Notify sender about errors:** Whether to notify senders about errors.
+    
+- **Error notification subject:** The subject to use for notifications about errors sent to senders.
+    
+- **Template for error notifications of sender:** A Smarty template to use for notifications about errors sent to senders.
+    
+- **Notification Activity Type:** The activity type to use for activities to notify senders about errors.
+    
+- **Archive Mailing:** Whether to archive mailings after sending. This feature is useful for preserving records of communications. This ensures that a copy of sent emails is available for future reference.
```

Comment:
Actually, if this option is disabled, mailings will be kept in the "Sent" status. Archiving moves them into the "Archived" list instead. This is not about "keep a copy".